src/main/java/be/syntra/devshop/DevshopFront/services/CartServiceImpl.java

```diff
@@ -74,4 +90,41 @@ private Product getProductFromCartById(Long productId) {
                 .findFirst()
                 .orElseThrow(() -> new ProductNotFoundException("Product with id = " + productId + " was not found in your cart"));
     }
+    @Override
+    public StatusNotification payCart(String userName) {
+        setUserName(currentCart, userName);
+        log.info("username() -> {}", currentCart.getUser());
```

Comment:
remove this log.info. A few lines below you're calling on the toString anyway, it's incorporated